spyder/plugins/plots/widgets/figurebrowser.py

```diff
@@ -481,8 +481,8 @@ def zoom_out(self):
 
     def scale_image(self):
         """Scale the image size."""
-        fwidth = self.figcanvas.fwidth
-        fheight = self.figcanvas.fheight
+        fwidth = self.figcanvas.fwidth * 1.0
```

Comment:
I think it should be more elegant to make use here `float` instead of multiplying by `1.0`. Also for all changes you made in this commit.

Reply:
@ccordoba12 Can we add `from __future__ import division` in this module instead to solve this for Python 2?

Reply:
Oh yeah, that's better! @gepcel, please do that instead.

Reply:
Good to learn about `from __future__ import division`